runtime/browser/web_application.cc

```diff
@@ -683,15 +652,15 @@ void WebApplication::OnRendered(WebView* /*view*/) {
   STEP_PROFILE_END("URL Set -> Rendered");
   STEP_PROFILE_END("Start -> Launch Completed");
   LOGGER(DEBUG) << "Rendered";
+  splash_screen_->HideSplashScreen();
```

Comment:
first-paint: The first visually non-empty paint has occurred.
  -> WebApplication::OnRendered
complete: All the resources have been loaded.
  -> WebApplication::OnLoadFinished
 user-interactive: The DOM and CSSOM have been constructed.
 -> WebApplication::OnLoadFinished

custom: The window.screen.show() method was called.
 -> Should be implemented.